test/functionalTest/cases/2015_notification_templates/notification_templates.test

```diff
@@ -337,7 +337,7 @@ User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
 Ngsiv2-Attrsformat: normalized
 Host: 127.0.0.1:REGEX(\d+)
 Accept: application/json
-Content-Type: application/json; charset=utf-8
+Content-Type: text/plain
```

Comment:
Regarding the utf-8 stuff, looking to the code we have:

```
  // For content-type application/json we add charset=utf-8
  if (orig_content_type == "application/json")
  {
    content_type += "; charset=utf-8";
  }
```

Not sure why we are doing that exception... maybe is obsolete (coming from the XML times) and should be applied to both defaults application/json and text/plain ?